pyccl/halos.py

```diff
@@ -0,0 +1,31 @@
+# Halo concentration
```

Comment:
As long as we are breaking the API, we should make this a legit submodule to clean up the code a bit. I realize this might break some of the swig build, but we should cross that bridge sooner rather than later maybe.

Reply:
OK, what would make this a legit submodule?

Reply:
You make a directory `pyccl/halos` and then put the code in files within that dir. Make sure to add an `__init__.py` to the directory. You will need to adjust the `setup.py` and any swig wrappers to make sure it builds right.

Reply:
ok, sounds good. I may ask for help with this, since I always get lost with `setup.py`